packages/open-payments/src/client/requests.ts

```diff
@@ -4,11 +4,12 @@ import { ResponseValidator } from 'openapi'
 import { BaseDeps } from '.'
 import { createSignatureHeaders } from './signatures'
 
-interface GetArgs {
+export interface GetArgs {
```

Comment:
I think we should keep these defined per resource, some routes have different `GetArgs`